CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ### Changed
 
+- Updated Jaeger Environment Variables (#1639)
```

Comment:
Please update this to include all end user relevant changes. This needs to include information about all behavior changes and anything that was exported being removed.

For information, please take a look at the linked [documentation](https://keepachangelog.com/en/1.0.0/).